app.lith.Lith.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "app.lith.Lith",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "Lith",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--talk-name=com.canonical.indicator.application",
+        "--talk-name=org.ayatana.indicator.application",
+        "--talk-name=org.freedesktop.portal.Fcitx",
+        "--own-name=org.kde.*",
```

Comment:
Trailing comma here makes JSON parser unhappy.

Reply:
Fixed, sorry about this.